CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 CHANGELOG for Sulu
 ==================
 
+* dev-master
+    * ENHANCEMENT #XXXX [SecurityBundle]    Made url for resetting password configurable via static variable
```

Comment:
Instead of XXXX it should say 1978 (that's the number of this PR). Will fix that for you.